.github/workflows/smoke-checks.yml

```diff
@@ -113,14 +116,14 @@ jobs:
         key: ${{ runner.os }}-mint-${{ hashFiles('./Mintfile') }}
         restore-keys: ${{ runner.os }}-mint-
     - uses: ./.github/actions/bootstrap
-    - name: Prepare iPhone 12 with iOS 15.0
+    - name: Prepare iPhone 12 with iOS 15.2
       uses: ./.github/actions/prepare-ios-simulator
       with:
-        device: "iPhone 12"
-        ios: "15.0"
-        xcode: "13.0"
+        device: ${{ env.IOS_SIMULATOR_DEVICE }}
+        ios: ${{ env.IOS_VERSION }}
+        xcode: ${{ env.XCODE_VERSION }}
```

Comment:
Do we really need to update the version for the UI Tests?

This will require re-recording all screenshots. If we do this for every version update the size of the repo will increase exponentially. So I would recommend updating the iOS Version for the UI Snapshot Tests very rarely. At least until we use something like Git LFS.

I'm also not sure if it is a good idea to use these environment variables because most of the time we need different version and environments for the tests 🤔

Reply:
Hi @nuno-vieira,

thanks for the comment.
I distilled 3 topics from your concerns:

1. Update of the Xcode version require us to everytime update the screenshots that are now part of the repo
- Yeah, this is definitely an issue, yet this issue is out of the scope of this PR. 
Altogether we updated cca 16 screenshots, where average size per screenshot is cca 6KB = +96KB in total. Is this a concern at this point in time?

2. Updating the iOS version requires us to re-record all the screenshots
- This is actually not the case. In this PR we got to update cca 16 screenshots generated by just a few snapshot tests.

3. _I'm also not sure if it is a good idea to use these environment variables because most of the time we need different version and environments for the tests_
If that was the case, the env variables can be overriden for such use cases. At this time they're shared across the actions. The added value of envs is that they're easily spotted and on top of the worfklow == easy to update across the stack if needed.

Reply:
@bielikb It depends on the iOS Versions. Some iOS Versions require more tests to be re-recorded. For every iOS update, this will add-up quite significantly

Reply:
@bielikb I'll approve the PR anyways. But I can forsee this being a problem in the future. We can revisit this in the future in case we see major differences between iOS Versions. 👍